tests/SalesOrdersClientTest.php

```diff
@@ -91,6 +91,43 @@ public function testCreateAndActivateTransferAppOrder()
         );
     }
 
+    public function testCreateAndActivateGoDaddyTransferOrder()
+    {
+        // Setup the client
+        $environment = getenv("ENVIRONMENT");
+        if ($environment == null) {
+            $environment = "DEMO";
+        }
+        $client = new SalesOrdersClient($environment);
+
+        // Create the request
+        $req = new CreateAndActivateOrderRequest();
+
+        // Create the line items
+        $goDaddy = SalesOrdersUtils::buildLineItem('MP-NNTJMBF6HPXR5XXC2JKCFWKJ64VZLBFQ');
+        $lineItems = array($goDaddy);
+
+        $zoneFile = "\$ORIGIN luckycharmsteam.com.\\r\\n\\r\\n; SOA Record\\r\\n@\\t3600\\t IN \\tSOA\\tns39.domaincontrol.com.\\tdns.jomax.net. (\\r\\n\\t\\t\\t\\t\\t2021060800\\r\\n\\t\\t\\t\\t\\t28800\\r\\n\\t\\t\\t\\t\\t7200\\r\\n\\t\\t\\t\\t\\t604800\\r\\n\\t\\t\\t\\t\\t3600\\r\\n\\t\\t\\t\\t\\t) \\r\\n\\r\\n; A Records\\r\\n@\\t3600\\t IN \\tA\\t104.154.100.138\\r\\n\\r\\n; CNAME Records\\r\\nwww\\t3600\\t IN \\tCNAME\\t@\\r\\n_domainconnect\\t3600\\t IN \\tCNAME\\t_domainconnect.ss.domaincontrol.com.\\r\\n\\r\\n; MX Records\\r\\n\\r\\n; TXT Records\\r\\n@\\t1800\\t IN \\tTXT\\t\\\"google-site-verification=wUvRFCYFmKdWGSkk82winL_D_tvO0TKUnP3cCpPhgo8\\\"\\r\\n\\r\\n; SRV Records\\r\\n\\r\\n; AAAA Records\\r\\n\\r\\n; CAA Records\\r\\n\\r\\n; NS Records\\r\\n@\\t3600\\t IN \\tNS\\tns39.domaincontrol.com.\\r\\n@\\t3600\\t IN \\tNS\\tns40.domaincontrol.com.\\r\\n";
```

Comment:
Since this is a public repo we don't want to include the actual zone file for this domain. It has the `google-site-verification` code and although this is just a test domain still good practice to not put that stuff in a repo.